openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmGCMXBeanNotifier.java

```diff
@@ -108,7 +110,8 @@ public void execute(StfCoreExtension test) throws StfException {
 		
 		// Process definition for the monitoring client JVM that connects with the server			
         JavaProcessDefinition client2JavaInvocation =  test.createJavaProcessDefinition()
-		    .addProjectToClasspath("openjdk.test.jlm")
+    		.addJvmOption("-Xmx256m")
+        	.addProjectToClasspath("openjdk.test.jlm")
```

Comment:
I think you have accidentally replaced the tab wth space here.

Reply:
Thanks! Corrected it.